docs/usecase/steps.md

```diff
@@ -160,6 +160,8 @@ There are a few reserved variables inside `ctx`:
 
 `ctx.req` - Contains the use case request payload. Declared when running a use case.
 
+`ctx.user` - Contains the user passed in authorization function of the use case.
```

Comment:
Suggested text:
Contains the information about the user authorized to run the use case. Declared at \[authorize](/docs/usecase/features#authorize) function.